pages/apply/[resident]/[[...step]].tsx

```diff
@@ -52,8 +53,14 @@ const ApplicationStep = (): JSX.Element => {
     }
   }
 
-  const onCompletion = () => {
-    // TODO: call API with patch
+  const onCompletion = async () => {
+    try {
+      // Below line currently returns a 404 as the endpoints arent ready yet
+      await updateApplication([currentResident])
```

Comment:
This is only the main applicant, I think we'd want to pass in all applicants similar to create scenario.